stubs/fpdf2/fpdf/html.pyi

```diff
@@ -58,6 +59,11 @@ class HTML2FPDF(HTMLParser):
     heading_above: float
     heading_below: float
     warn_on_tags_not_matching: bool
+
+    # Not initialized in __init__:
+    font_face: Incomplete
+    h: Incomplete
```

Comment:
It's whatever is returned by `px2mm`: https://github.com/PyFPDF/fpdf2/blob/60f5104e9a125d2ac0288a23960d0b1960e54329/fpdf/html.py#L574

And `px2mm` returns a float: https://github.com/PyFPDF/fpdf2/blob/60f5104e9a125d2ac0288a23960d0b1960e54329/fpdf/html.py#L177-L178

```suggestion
    h: float
```